test/nodes/test_file_converter.py

```diff
@@ -17,14 +16,10 @@
 
 
 @pytest.mark.tika
-@pytest.mark.parametrize(
-    # "Converter", [PDFToTextConverter, TikaConverter, PDFToTextOCRConverter]
-    "Converter",
-    [PDFToTextOCRConverter],
-)
+@pytest.mark.parametrize("Converter", [PDFToTextConverter, TikaConverter, PDFToTextOCRConverter])
 def test_convert(Converter):
```

Comment:
We could add a docstring in this test mentioning 
> If you are running the test_convert test locally, make sure that poppler is installed or otherwise the pdf will be converted into an empty list of images by PDFToTextOCRConverter.

or even fail explicitly at runtime with a message about the missing requirement if possible